src/Microsoft.AspNet.Security.Google/GoogleAuthenticationHandler.cs

```diff
@@ -34,45 +34,45 @@ public GoogleAuthenticationHandler(HttpClient httpClient, ILogger logger)
 
             var context = new GoogleAuthenticatedContext(Context, Options, user, tokens);
             context.Identity = new ClaimsIdentity(
```

Comment:
Instead of creating a `ClaimsIdentity` and wrapping it with a `ClaimsPrincipal` in several places like `AuthenticationTicket` can we also new up a `Principal` in the middleware itself so that we have a consistent implementation. All user facing contexts now contain Principal anyway. I think Wilson produces `Principal` now.